qt/scientific_interfaces/Indirect/IIndirectFitRegion.h

```diff
@@ -1,42 +0,0 @@
-// Mantid Repository : https://github.com/mantidproject/mantid
-//
-// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
-//   NScD Oak Ridge National Laboratory, European Spallation Source,
-//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
-// SPDX - License - Identifier: GPL - 3.0 +
-#pragma once
-#include <utility>
-
-#include "DllConfig.h"
-#include "IndexTypes.h"
-
-namespace MantidQt {
-namespace CustomInterfaces {
-namespace IDA {
-
-/*
-    IIndirectFitData - Specifies an interface for updating, querying and
-   accessing the raw data in IndirectFitAnalysisTabs
-*/
-class MANTIDQT_INDIRECT_DLL IIndirectFitRegion {
```

Comment:
Should this be added back in?

Reply:
This is not actually used in any capacity in the code base.